lib/components/AlbumScreen/song_menu.dart

```diff
@@ -170,6 +180,62 @@ class _SongMenuState extends State<SongMenu> {
     }
   }
 
+  Future<bool> shouldShowSpeedWidget(currentSpeed) async {
```

Comment:
There is no `SpeedWidget` (it's a button and a menu). Maybe call it `SpeedControl(s)` instead?